storage/bucket.go

```diff
@@ -462,6 +464,14 @@ type BucketAttrs struct {
 	// allows for the automatic selection of the best storage class
 	// based on object access patterns.
 	Autoclass *Autoclass
+
+	// ObjectRetentionMode reports whether individual objects in the bucket can
+	// be configured with a retention policy. An empty value means that object
+	// retention is disabled. Object retention is not enabled default.
+	// This field is read-only. Object retention can be enabled only on bucket
```

Comment:
Read-only seems not really correct given that it can be set on bucket creation, but I think you can just delete that sentence and the rest of the doc is fine.